test/classes/diten/multipledestructor.chpl

```diff
@@ -1,7 +1,7 @@
 class C {
-  proc ~C() { writeln("In ~C #1"); }
+  proc deinit() { writeln("In deinit #1"); }
 }
-proc C.~C() { writeln("In ~C #2"); }
+proc C.deinit() { writeln("In deinit #2"); }
```

Comment:
It involves deinitializers now, which is directly related to initializers.  If my work happens to modify deinitializers, I would discover this through running both the aforementioned directories and if necessary would update those tests.  Having this test outside of those directories would mean that I wouldn't necessarily remember this one and so wouldn't include it in the commit with the others and wouldn't discover the change until the full standard run.  We should as much as possible avoid spreading tests related to a feature out in general for this very reason.  (btw, I'm stating my supporting viewpoints so you understand where I'm coming from, not because I necessarily think you disagree with them)